package.json

```diff
@@ -4,7 +4,8 @@
   "description": "The application to test is ToolsQA Website that can be found and accessed at: https://demoqa.com/",
   "main": "index.js",
   "scripts": {
-    "test": "wdio wdio.conf.js"
+    "test": "wdio wdio.conf.js",
+    "report": "npx generate allure --clean && allure open"
```

Comment:
Update this command by adding the path to allure-results folder.

Reply:
I did after commit and the report still doesn't generate